deploy-manage/monitor/autoops/ec-autoops-deployment-view.md

```diff
@@ -1,21 +1,22 @@
 ---
-navigation_title: Deployment
+navigation_title: Deployment or Cluster
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud/current/ec-autoops-deployment-view.html
 applies_to:
   deployment:
     ess: all
```

Comment:
same question as above about whether it should be `ess: all` or just `ess:` (but probably I just don't understand the tagging properly).